webpack/env.js

```diff
@@ -0,0 +1,31 @@
+const nodeEnv = process.env.NODE_ENV;
+
+const development = 'development';
+const production = 'production';
+const prodStrings = [
+  'prd',
+  'prod',
+  'production',
```

Comment:
Is it necessary to account for all of these cases when `package.json` only uses `"prod"`?

Reply:
I think there is still value in providing these options for ease of use - they can be documented in completing Issue #29 as convenient ways to run locally for testing/debug purposes, ex: `NODE_ENV=prd npx webpack --config webpack/contrived-test-config.js`

Reply:
Rationale feels a bit flimsy to me, but I don't see the harm...